### ISSUE TITLE
Was working, now  : the requested URL is forbidden.

### ISSUE THREAD

**Issue by rileyrg** [jcs-elpa/impatient-showdown#1]
Firstly: great package. 
But unfortunately, it has stopped working and I can't pinpoint why. A clue as to why the pushed URL is now illegal?

http://localhost:8080/imp/live/README.md/ 

> 
> Forbidden
> The requested URL is forbidden.
> 
> error: Buffer README.md is private or doesn't exist.

.

**Comment by jcs090218** [comment]
Hi @rileyrg, I am not able to reproduce this issue. Can you try setting the variable `debug-on-error` to `t` and see the backtrace? I think this issue is either cause by the upstream [impatient-mode](https://github.com/skeeto/impatient-mode) or due to the unclose http server. 😕

**Comment by rileyrg** [comment]
There is no backtrace I'm afraid.  Could it be linked to the fact that I'm exporting from a xxx.org to README.md?

https://github.com/rileyrg/el-docstring-sap

**Comment by jcs090218** [comment]
I don't think that's the culprit, but I would recommend you report this bug to the upstream package `impatient-mode` since I have encountered this issue too but without any method reproducing it. From what I remembered, once you restart Emacs then everything should be resolved?

**Comment by rileyrg** [comment]
No, I turned it off and on again. ;) I'll raise it upstream soon. Thanks for your reply.
